src/CPE/public/cpe_name.h

```diff
@@ -290,16 +290,6 @@ OSCAP_API bool cpe_name_set_other(struct cpe_name *cpe, const char *newval);
  */
 OSCAP_API bool cpe_name_match_one(const struct cpe_name *cpe, const struct cpe_name *against);
 
-/**
- * Check if CPE @a name matches any CPE in @a namelist.
- * @memberof cpe_name
- * @param name name to be looked-up
- * @param n number of items in namelist
- * @param namelist list of names to search in
- * @return true if @a name was found within @a namelist
- */
-OSCAP_API bool cpe_name_match_cpes(const struct cpe_name *name, size_t n, struct cpe_name **namelist);
```

Comment:
ACK. Completely removed and doesn't look like it is used externally:

https://github.com/search?q=language%3Ac+cpe_name_match_cpes&type=Code